apps/charterafrica/src/payload/fields/ecosystem/sourceField.js

```diff
@@ -0,0 +1,37 @@
+import { deepmerge } from "@mui/utils";
+
+import airtableColumnSelect from "./airtableColumnSelect";
+
+function sourceField({ tableField, overrides = {} }) {
+  const defaultField = {
+    type: "group",
+    label: { en: "Source", fr: "Source", pt: "Fonte" },
+    name: "source",
+    fields: [
+      airtableColumnSelect({
+        tableField,
+        overrides: {
+          name: "url",
+          label: { en: "Source URL", fr: "URL de source", pt: "URL da fonte" },
+        },
+      }),
+      airtableColumnSelect({
+        tableField,
+        overrides: {
+          name: "type",
```

Comment:
Do we have the type column in the Airtable? I'm assuming this will be github, gitlab, etc., and we only support github for now.

Reply:
@kilemensi  yes.

Can either be
For-Profit, Non-Profit, Academics & Research etc.

So we can't fetch this from github

Reply:
Unless I'm missing something, I'm asking about `source` type @koechkevin. Not `organisation` type.

Reply:
@kilemensi.
My thinking is,

There will be 1 Airtable base to collect this data.
So data team will input all the data in the tables then have a column tthat specifies source of data in each row

Reply:
I had asked Keziah to add this column.